### ISSUE TITLE
NOT Update a typo in bootstrap.sh on ami-07fdc9272ce5b0ce5

### ISSUE THREAD

**Issue by ffjlabo** [awslabs/amazon-eks-ami#229]
**What happened**:
When I created EC2 instance using `ami-07fdc9272ce5b0ce5` and tried to execute `/etc/eks/bootstrap.sh --enable-docker-bridge <this is my clustername>`, It didn't work well. 
 
```
sudo /etc/eks/bootstrap.sh <my cluster name> --enable-docker-bridge
/etc/eks/bootstrap.sh: 行 56: $2: 未割り当ての変数です
```
So I read the file and found the reason.
There is a typo fixed by https://github.com/awslabs/amazon-eks-ami/commit/613fecec8c30c4dfb73675da1eedb079f9889c9a#diff-d53a8bed778fd70d6478cd9b0981810f in `/etc/eks/bootstrap.sh` but the fix isn't reflected in it on `ami-07fdc9272ce5b0ce5`.

This is because that a PR for it(#192) is merged on Feb 20, but the ami is published on Feb 19.
<img width="994" alt="AMIs___EC2_Management_Console" src="https://user-images.githubusercontent.com/40124947/54870161-3778ef00-4de6-11e9-91c2-3f0a920fc2aa.png">

So could you update the ami ?
Sorry for my bad English...

PS. Below is `/etc/eks/bootstrap.sh` in my instance.
``` shell
#!/usr/bin/env bash

set -o pipefail
set -o nounset
set -o errexit

err_report() {
    echo "Exited with error on line $1"
}
trap 'err_report $LINENO' ERR

IFS=$'\n\t'

function print_help {
    echo "usage: $0 [options] <cluster-name>"
    echo "Bootstraps an instance into an EKS cluster"
    echo ""
    echo "-h,--help print this help"
    echo "--use-max-pods Sets --max-pods for the kubelet when true. (default: true)"
    echo "--b64-cluster-ca The base64 encoded cluster CA content. Only valid when used with --apiserver-endpoint. Bypasses calling \"aws eks describe-cluster\""
    echo "--apiserver-endpoint The EKS cluster API Server endpoint. Only valid when used with --b64-cluster-ca. Bypasses calling \"aws eks describe-cluster\""
    echo "--kubelet-extra-args Extra arguments to add to the kubelet. Useful for adding labels or taints."
    echo "--enable-docker-bridge Restores the docker default bridge network. (default: false)"
}

POSITIONAL=()

while [[ $# -gt 0 ]]; do
    key="$1"
    case $key in
        -h|--help)
            print_help
            exit 1
            ;;
        --use-max-pods)
            USE_MAX_PODS="$2"
            shift
            shift
            ;;
        --b64-cluster-ca)
            B64_CLUSTER_CA=$2
            shift
            shift
            ;;
        --apiserver-endpoint)
            APISERVER_ENDPOINT=$2
            shift
            shift
            ;;
        --kubelet-extra-args)
            KUBELET_EXTRA_ARGS=$2
            shift
            shift
            ;;
        --enable-docker-bridge)
            ENABLE_DOCKER_BIDGE=$2
            shift
            shift
            ;;
        *)    # unknown option
            POSITIONAL+=("$1") # save it in an array for later
            shift # past argument
            ;;
    esac
done

set +u
set -- "${POSITIONAL[@]}" # restore positional parameters
CLUSTER_NAME="$1"
set -u

USE_MAX_PODS="${USE_MAX_PODS:-true}"
B64_CLUSTER_CA="${B64_CLUSTER_CA:-}"
APISERVER_ENDPOINT="${APISERVER_ENDPOINT:-}"
KUBELET_EXTRA_ARGS="${KUBELET_EXTRA_ARGS:-}"
ENABLE_DOCKER_BRIDGE="${ENABLE_DOCKER_BRIDGE:-false}"

if [ -z "$CLUSTER_NAME" ]; then
    echo "CLUSTER_NAME is not defined"
    exit  1
fi

ZONE=$(curl -s http://169.254.169.254/latest/meta-data/placement/availability-zone)
AWS_DEFAULT_REGION=$(echo $ZONE | awk '{print substr($0, 1, length($0)-1)}')

### kubelet kubeconfig

CA_CERTIFICATE_DIRECTORY=/etc/kubernetes/pki
CA_CERTIFICATE_FILE_PATH=$CA_CERTIFICATE_DIRECTORY/ca.crt
mkdir -p $CA_CERTIFICATE_DIRECTORY
if [[ -z "${B64_CLUSTER_CA}" ]] && [[ -z "${APISERVER_ENDPOINT}" ]]; then
    DESCRIBE_CLUSTER_RESULT="/tmp/describe_cluster_result.txt"
    aws eks describe-cluster \
        --region=${AWS_DEFAULT_REGION} \
        --name=${CLUSTER_NAME} \
        --output=text \
        --query 'cluster.{certificateAuthorityData: certificateAuthority.data, endpoint: endpoint}' > $DESCRIBE_CLUSTER_RESULT
    B64_CLUSTER_CA=$(cat $DESCRIBE_CLUSTER_RESULT | awk '{print $1}')
    APISERVER_ENDPOINT=$(cat $DESCRIBE_CLUSTER_RESULT | awk '{print $2}')
fi

echo $B64_CLUSTER_CA | base64 -d > $CA_CERTIFICATE_FILE_PATH

sed -i s,CLUSTER_NAME,$CLUSTER_NAME,g /var/lib/kubelet/kubeconfig
kubectl config \
    --kubeconfig /var/lib/kubelet/kubeconfig \
    set-cluster \
    kubernetes \
    --certificate-authority=/etc/kubernetes/pki/ca.crt \
    --server=$APISERVER_ENDPOINT

### kubelet.service configuration

MAC=$(curl -s http://169.254.169.254/latest/meta-data/network/interfaces/macs/ -s | head -n 1)
CIDRS=$(curl -s http://169.254.169.254/latest/meta-data/network/interfaces/macs/$MAC/vpc-ipv4-cidr-blocks)
TEN_RANGE=$(echo $CIDRS | grep -c '^10\..*' || true )
DNS_CLUSTER_IP=10.100.0.10
if [[ "$TEN_RANGE" != "0" ]] ; then
    DNS_CLUSTER_IP=172.20.0.10;
fi

KUBELET_CONFIG=/etc/kubernetes/kubelet/kubelet-config.json
echo "$(jq ".clusterDNS=[\"$DNS_CLUSTER_IP\"]" $KUBELET_CONFIG)" > $KUBELET_CONFIG

INTERNAL_IP=$(curl -s http://169.254.169.254/latest/meta-data/local-ipv4)
INSTANCE_TYPE=$(curl -s http://169.254.169.254/latest/meta-data/instance-type)

if [[ "$USE_MAX_PODS" = "true" ]]; then
    MAX_PODS_FILE="/etc/eks/eni-max-pods.txt"
    set +o pipefail
    MAX_PODS=$(grep ^$INSTANCE_TYPE $MAX_PODS_FILE | awk '{print $2}')
    set -o pipefail
    if [[ -n "$MAX_PODS" ]]; then
        echo "$(jq ".maxPods=$MAX_PODS" $KUBELET_CONFIG)" > $KUBELET_CONFIG
    else
        echo "No entry for $INSTANCE_TYPE in $MAX_PODS_FILE. Not setting max pods for kubelet"
    fi
fi

cat <<EOF > /etc/systemd/system/kubelet.service.d/10-kubelet-args.conf
[Service]
Environment='KUBELET_ARGS=--node-ip=$INTERNAL_IP --pod-infra-container-image=602401143452.dkr.ecr.$AWS_DEFAULT_REGION.amazonaws.com/eks/pause-amd64:3.1'
EOF

if [[ -n "$KUBELET_EXTRA_ARGS" ]]; then
    cat <<EOF > /etc/systemd/system/kubelet.service.d/30-kubelet-extra-args.conf
[Service]
Environment='KUBELET_EXTRA_ARGS=$KUBELET_EXTRA_ARGS'
EOF
fi

if [[ "$ENABLE_DOCKER_BRIDGE" = "true" ]]; then
    # Enabling the docker bridge network. We have to disable live-restore as it
    # prevents docker from recreating the default bridge network on restart
    jq '.bridge="docker0" | ."live-restore"=false' /etc/docker/daemon.json | tee  /etc/docker/daemon.json
    systemctl restart docker
fi

systemctl daemon-reload
systemctl enable kubelet
systemctl start kubelet
```

**What you expected to happen**:
I want you to recreate the ami. And I want to use `--enable-docker-bridge` option.

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- AWS Region: ap-northeast-1
- Instance Type(s): c5.large
- EKS Platform version (use `aws eks describe-cluster --name <name> --query cluster.platformVersion`):"eks.2"
- Kubernetes version (use `aws eks describe-cluster --name <name> --query cluster.version`):"1.11"
- AMI Version:ami-07fdc9272ce5b0ce5
- Kernel (e.g. `uname -a`):
- Release information (run `cat /tmp/release` on a node):

```
```

**Comment by sc250024** [comment]
I'm seeing the same thing on the `ami-098fb7e9b507904e7)` AMI in eu-west-1 region. The typo is the culprit indeed:

```bash
 55         --enable-docker-bridge)
 56             ENABLE_DOCKER_BIDGE=$2
 57             shift
 58             shift
 59             ;;
 60         *)    # unknown option
 61             POSITIONAL+=("$1") # save it in an array for later
 62             shift # past argument
 63             ;;
 64     esac
 65 done
 66
 67 set +u
 68 set -- "${POSITIONAL[@]}" # restore positional parameters
 69 CLUSTER_NAME="$1"
 70 set -u
 71
 72 USE_MAX_PODS="${USE_MAX_PODS:-true}"
 73 B64_CLUSTER_CA="${B64_CLUSTER_CA:-}"
 74 APISERVER_ENDPOINT="${APISERVER_ENDPOINT:-}"
 75 KUBELET_EXTRA_ARGS="${KUBELET_EXTRA_ARGS:-}"
 76 ENABLE_DOCKER_BRIDGE="${ENABLE_DOCKER_BRIDGE:-false}"
```

The exact error when the `--enable-docker-bridge` argument is passed is:

```shell
sed: -e expression #1, char 88: unknown option to `s'
Exited with error on line 104
```

**Comment by mogren** [comment]
Yes, this typo was fixed in #192. New worker AMIs will come really soon. Tracked here: https://github.com/aws/containers-roadmap/issues/188#issuecomment-474010632
